README.md

```diff
@@ -1,3 +1,22 @@
+
+
+
+<p align="center">
+    <img src="https://img.shields.io/github/workflow/status/mini-services/miniurl/MiniUrl" alt="Build">
+    <img src="https://img.shields.io/github/workflow/status/mini-services/miniurl/Test?label=test" alt="Test">
+    <a href="https://codecov.io/gh/mini-services/miniurl"><img src="https://codecov.io/gh/mini-services/miniurl/branch/release/0.2.0/graph/badge.svg?token=YE7LE819HK" alt="Codecov"></a>
```

Comment:
Although it doesn't show anything yet, this should be directed to the `master` branch (seems like it's currently on `release/0.2.0`.

Reply:
The change was made